Pegasus/include/Math.hpp

```diff
@@ -18,11 +18,87 @@
 #include <set>
 #include <unordered_map>
 #include <unordered_set>
+#include <numeric>
 
 namespace pegasus
 {
 namespace math
 {
+namespace fp
+{
+constexpr double g_floatingPointThreshold = 1e-4;
+
+/**
+ * @brief Calculates whether a given floating point number is equal to null within a set threshold
+ * @param[in] n number
+ * @return @c true if a number is equal to null, @c false otherwise
+ */
+inline bool IsNull(double n)
```

Comment:
Well, I personally am used to "null" meaning "null pointer in C++. "Zero" is unambiguously a number. Personal taste though.